recipes/jill/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "jill" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jill-{{ version }}.tar.gz
+    sha256: 48ea86a508f87dab0293d00634b4e8c6677cb00676e617b6c46345bc10af5e10
+  #- url: https://raw.githubusercontent.com/johnnychen94/jill.py/master/requirements.txt
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    ## Ensure inclusion of requirements.txt
+    {% set reqfile = "requirements.txt" %}
+    {% set reqsource = name ~ ".egg-info/requires.txt" %}
+    # IF {{ reqfile }} is NOT PRESENT, COPY {{ reqfile }} from {{ reqsource }}
+    - echo -e "\n>>> ENSURE INCLUSION OF '{{ reqfile }}' <<<\n\n"
+    - if [ ! -f {{ reqfile }} ] && [ -f {{ reqsource }} ]; then cp {{ reqsource }} {{ reqfile }}; fi  # [not win]
+    #- copy {{ name }}.egg-info\requires.txt {{ reqfile }}  # [win]
+    - copy {{ reqsource | replace('/', '\\') }} {{ reqfile }}  # [win]
+    ## Install package
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - wget
+  run:
+    - python >=3.6
+    - fire
+    - jsonschema
+    - python-gnupg
+    - requests
+    - requests-futures
+    - semantic_version
+    - wget
+
+test:
+  imports:
+    - jill
+  # commands:
+  #   - pip check
+  # requires:
+  #   - pip
+  #   # wget is necessary for passing pip-check
+  #   # despite adding it under run-requirements.
+  #   # - wget
```

Comment:
Are you sure? Anything under run is available during testing.

Reply:
Had tried with it uncommented. But it was failing. It did not matter where I added wget. It would still fail in the pip-check step. So, got it commented to pass it. 

But did not remove any of the comments, only to see if the review provides some better alternatives. 

Just to be clear, in the worse scenario, I will remove all inclusions of wget from the recipe and remove pip-check as well.

Reply:
😄 

Ahh ok. I would suggest adding all of the necessary requirements specified by upstream (If that includes wget, it should be added as well) and then removing pip check.